### ISSUE TITLE
Custom card

### ISSUE THREAD

**Issue by 22MarioZ** [r-renato/hass-xiaomi-mi-flora-and-flower-care#4]
Where can I get the custom card for lovelace as is not installed with the integration.

resources:
  - url: /local/hass-xiaomi-mi-flora-and-flower-care/hass-xiaomi-mi-flora-and-flower-care.js
    type: js

The Integration works well.

**Comment by ggmanugg** [comment]
https://github.com/r-renato/hass-xiaomi-mi-flora-and-flower-care#manual-installation
or via [HACS](https://hacs.xyz/)

**Comment by 22MarioZ** [comment]
Dear,

I used HACS but the *.js for the lovelace card did not install. I'm just following the instructions hoe to use the custom card in lovelace:
`type: custom:xiaomi-mi-flora-and-flower-care-card`
`name:` "Zamioculcas `Zamiifolia"`
`entity:` plant.plant_zamioculcas_zamiifolia`

**Comment by ggmanugg** [comment]
Aah yes i have the same Issue on my installation. The expected folder is nowhere.
But since I'm using the the [Mi Flora card](https://www.home-assistant.io/integrations/miflora/), I didn't noticed it.

**Comment by mvheimburg** [comment]
I installed with HACS but did neither get the .js or the custom_components folder. Folder is fixed by manual installation, but how can I fix the .js?

**Comment by r-renato** [comment]
Hi, I have update the custom component and fixed this.

**Comment by lewiatanx** [comment]
Hello, after installing and configuring, only such a panel appears to me:
What could be wrong ???
![kwiatek](https://user-images.githubusercontent.com/61457646/76503146-998e9200-6445-11ea-8fff-993231add17b.jpg)

**Comment by grittisemper** [comment]
Hi,
this can be a configuration problem.
Check if your configuration is ok or put it here if you want help.

**Comment by lewiatanx** [comment]
HI **grittisemper** :)
I checked the configuration and it's probably ok.
The data displayed from the sensor works on the built-in plug-in while there is nothing through huahuacacao.
The attached file describes the configuration.
I have no idea where the problem is ...
[huahuacaocao.txt](https://github.com/r-renato/hass-xiaomi-mi-flora-and-flower-care/files/4324796/huahuacaocao.txt)
![lovelace](https://user-images.githubusercontent.com/61457646/76533810-edff3500-6478-11ea-82d4-6fd523f6ba41.jpg)

**Comment by r-renato** [comment]
hi,
I think should be `"draceana arborea"` the right **plant_id**, not `"dracena"`.

**Comment by lewiatanx** [comment]
hi :)
I removed the flower from the mobile application, I configured the sensor from the beginning. Now the flower is "dracaena fragrans". I have corrected entries in the hassio configuration and now something is wrong, but the description has already appeared, but there is no information from the sensor. What could be the reason?
![lovelace2](https://user-images.githubusercontent.com/61457646/76546761-573c7380-648c-11ea-8979-3e636c4ee3dd.jpg)
